src/__tests__/components/ExchangeQuoteComponent.test.js

```diff
@@ -0,0 +1,31 @@
+/* globals describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import * as React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { ExchangeQuoteComponent } from '../../components/themed/ExchangeQuoteComponent.js'
+
+describe('ExchangeQuote', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      isTop: true | undefined,
```

Comment:
This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.